labels.yaml

```
@@ -2,4 +2,473 @@ orgs:
 - meltano
 - MeltanoLabs
 
-labels: []
+labels:
+  ####################
+  #      Colors      #
+  ####################
+  # red:            "B60205"
+  # orange:         "D93F0B"
+  # yellow:         "FBCA04"
+  # green:          "0E8A16"
+  # teal:           "006B75"
+  # blue:           "1D76DB"
+  # navy:           "0052CC"
+  # purple:         "5319E7"
+  # light_red:      "E99695"
+  # light_orange:   "F9D0C4"
+  # light_yellow:   "FEF2C0"
+  # light_green:    "C2E0C6"
+  # light_teal:     "BFDADC"
+  # light_blue:     "C5DEF5"
+  # light_teal:     "BFD4F2"
+  # light_purple:   "D4C5F9"
+
+  ####################
+  #      Common      #
+  ####################
+  - name: "Documentation"
+    description: Improvements or additions to documentation
+    color: "0075CA"
```

We should be consistent with our tag capitalization. I think all-lowercase is appropriate, maybe with a few exceptions.

```suggestion
  - name: "documentation"
    description: Improvements or additions to documentation
    color: "0075CA"
```

---

I won't suggest lowercasing the other uppercase letters in tags until we decide how we should handle this.